src/test/java/org/pentaho/metadata/automodel/PhysicalTableImporterTest.java

```diff
@@ -30,9 +30,9 @@
 import org.pentaho.metadata.model.SqlPhysicalTable;
 
 import java.util.List;
+import java.util.Map;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
Yeah... I most rules we only get the **what* not the **why**, so it's harder to generalize in new cases. If you have that impression and have been using it, i'm ok with it.